packages/react-query-devtools/src/Explorer.tsx

```diff
@@ -124,6 +155,9 @@ export const DefaultRenderer: Renderer = ({
               {subEntries.length} {subEntries.length > 1 ? `items` : `item`}
             </Info>
           </ExpandButton>
+          {copiable ? (<CopyButton>
+            <Copier onClick={() => navigator.clipboard.writeText(displayValue(value))} />
```

Comment:
putting `displayValue` into the clipboard seems weird, because we use `superjson` to show non-serializable values. Why don't we just put the raw values that are really in the cache into the clipboard?